pyanaconda/payload/__init__.py

```diff
@@ -472,6 +472,7 @@ def addRepo(self, ksrepo):
         # Add the repo to the ksdata so it'll appear in the output ks file.
         self.data.repo.dataList().append(ksrepo)
 
+    # TODO: this function is never used
     def removeRepo(self, repo_id):
```

Comment:
I think this functionality may stay here.

Reply:
Kept.